geomstats/geometry/matrices_space.py

```diff
@@ -62,6 +62,23 @@ def commutator(a, b):
         """
         return gs.matmul(a, b) - gs.matmul(b, a)
 
+    @staticmethod
+    def transpose(mat):
+        """
+        Return the transpose of matrices.
+
+        Parameters
+        ----------
+        mat : array-like, shape=[n_samples, dim, dim]
+
+        Returns
+        -------
+        transpose : array-like, shape=[n_samples, dim, dim]
+        """
+        is_vec = (gs.ndim(gs.array(mat)) == 3)
```

Comment:
NIT: call this `is_vectorized`, in order not to be confused with `is_vector`?

Reply:
ok fair enough... 
Je vais changer le nom dans mon master, mais laisse-moi le push dans une future PR stp! ... j'ai encore fait évoluer ma branche avec grassmann.log et c'est suffisament mineur pour que je ne me rebatte pas avec `git cherry-pick` ^^